<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="logFileSourcesTab">
      <attribute name="title">
       <string>Log file sources</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_13">
         <property name="title">
          <string>Your log file sources</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_21">
          <item row="0" column="0">
           <widget class="QFrame" name="logFileSourceSelectFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_20">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>9</number>
             </property>
             <item row="0" column="0">
              <widget class="QListWidget" name="logFileSourceListWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>2</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_23">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QPushButton" name="logFileSourceAddButton">
                  <property name="text">
                   <string>&amp;Add source</string>
                  </property>
                  <property name="icon">
                   <iconset theme="folder-new" resource="../media.qrc">
                    <normaloff>:/media/folder-new.svg</normaloff>:/media/folder-new.svg</iconset>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="logFileSourceRemoveButton">
                  <property name="text">
                   <string>&amp;Remove source</string>
                  </property>
                  <property name="icon">
                   <iconset theme="edit-delete" resource="../media.qrc">
                    <normaloff>:/media/edit-delete.svg</normaloff>:/media/edit-delete.svg</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QFrame" name="logFileSourceEditFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_22">
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="logFileSourceOwnCloudServerLabel">
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="logFileSourceActiveCheckBox">
               <property name="text">
                <string>Use as active source</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLineEdit" name="logFileSourceLocalPathLineEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Local folder path</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QPushButton" name="logFileSourceLocalPathButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="document-open" resource="../media.qrc">
                 <normaloff>:/media/document-open.svg</normaloff>:/media/document-open.svg</iconset>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QComboBox" name="logFileSourceTypeComboBox">
               <item>
                <property name="text">
                 <string>Local</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>eZ Publish Server</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLineEdit" name="logFileSourceNameLineEdit">
               <property name="toolTip">
                <string>This name will be viewed in all the menus.</string>
               </property>
               <property name="placeholderText">
                <string>a name you will recognize</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0" colspan="2">
              <spacer name="verticalSpacer_9">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="8" column="0" colspan="2">
              <widget class="QStackedWidget" name="logFileSourceTypeConfigStackedWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="logFileSourceTypeLocalPage">
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                </layout>
               </widget>
               <widget class="QWidget" name="logFileSourceTypeEzPublishPage">
                <layout class="QGridLayout" name="gridLayout_4">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="5" column="0">
                  <widget class="QLineEdit" name="logFileSourceEzPublishServerPasswordLineEdit">
                   <property name="toolTip">
                    <string>Set the ownCloud path where the note folder path will be synced to on your ownCloud server.</string>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string notr="true">your password</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>Password</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>eZ Publish Server URL</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="logFileSourceEzPublishServerUrlLineEdit">
                   <property name="toolTip">
                    <string>Set the ownCloud path where the note folder path will be synced to on your ownCloud server.</string>
                   </property>
                   <property name="placeholderText">
                    <string notr="true">https://your-server-name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>Username</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLineEdit" name="logFileSourceEzPublishServerUsernameLineEdit">
                   <property name="toolTip">
                    <string>Set the ownCloud path where the note folder path will be synced to on your ownCloud server.</string>
                   </property>
                   <property name="placeholderText">
                    <string notr="true">your username</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QCheckBox" name="logFileSourceAddDownloadedFilePrefixCheckBox">
                   <property name="toolTip">
                    <string>This is usefull when downloading from multiple log file sources to the same local directory</string>
                   </property>
                   <property name="text">
                    <string>Add a prefix to downloaded files</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../media.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
